morph_tool/nrnhines.py

```diff
@@ -278,7 +278,19 @@ class NestedPool(multiprocessing.pool.Pool):  # pylint: disable=abstract-method
 
     class Process(multiprocessing.Process):
         """Class that represents a non-daemon process"""
-        daemon = False
+
+        def __init__(self, *args, **kwargs):
+            """Ensures group=None."""
+            kwargs.pop("group", None)
```

Comment:
```python
if 'group' in kwargs
```
looks clearer as it does not require a comment that explains this adhoc.

Reply:
But is would perform 2 searches in the kwargs dict instead of one. Isn't it clear enough like this?

Reply:
readability must come first

Reply:
Well, I would say it depends :-)
But this method is not supposed to be called millions of times so let's change it.

Reply:
In the comment it's better to explain why `group` must be set to `None`

Reply:
I added a short explanation